src/integration-test/helpers/clients/idamClient.ts

```diff
@@ -4,6 +4,12 @@ const baseURL: string = process.env.IDAM_URL
 
 const defaultPassword = 'Password12'
 
+const oauth2 = {
+  client_id: 'cmc_citizen',
+  redirect_uri: 'https://localhost:3000/receiver',
```

Comment:
the redirect_uri and client_secret need to be brought in as environment variables like @sabahirfan did in the claim-store change